gemfirexd/tools/src/main/java/com/pivotal/gemfirexd/internal/impl/tools/ij/utilMain.java

```diff
@@ -765,8 +765,15 @@ private boolean doCatch(String command) {
                         else {
                           redirected = null;
                         }
+// GemStone changes BEGIN
+      boolean displayCount = false;
+      if (command.startsWith("insert") || command.startsWith("update")
+        || command.startsWith("delete") || command.startsWith("put") ) {
+        displayCount = true;
+      }
```

Comment:
What if there are comments, spaces or other tokens? Why not add the flag in ijParser as per the token as mentioned above. Or use ClientSharedUtils.getStatementToken() and then check on the returned token which should be much more reliable.